tests/xrt/perf_IOPS/xilutil.hpp

```diff
@@ -0,0 +1,59 @@
+/* Xilinx XRT test case utility header file */
+
+#ifndef XIL_UTIL_HPP
+#define XIL_UTIL_HPP
+
+#include <mutex>
+#include <condition_variable>
+
+/* std::barrier can be used since c++20
+ * Thanks boost library. Modified a little bit with pure c++11 code
+ */
+class barrier
```

Comment:
Why not using Boost (or C++20...) at the first place instead of reimplementing libraries?

Reply:
Not every compiler can compile c++20. We have a machine that cannot compile c++14.